lib/core/datatable/components/datatable/datatable.component.scss

```diff
@@ -18,8 +18,15 @@ $data-table-cell-min-width--fileSize: $data-table-cell-min-width !default;
     height: 100%;
 
     .adf-full-width {
+        min-width: 800px;
```

Comment:
800 px works for us because we have 4 columns by default in ADW. What if a customer has 8 columns for example? Will this fix work?

Reply:
from a quick check I did it won't work, can you double check please?

Reply:
done the changes for dynamic number of columns, please review